### PR TITLE
Tests refactoring part 1

### PR DESCRIPTION
**Pull request by dataclouder**
This is the first part of the tests refactoring (Issue#10) to get rid of the mock server in tests.
To simplify the review, I committed each file separately.

The purpose of this task (and a similar one by @auppunda with the remaining tests) is to make the test run with a real vCD, replicating the behavior of the previous test. 
There are more tests that can be created and modified, but that will come later, once the foundation for the test suite is completed.

A general note on the code: the test suite used `check.v1`, which recommends some odd behavior, like not using a name space for the testing library and squash the function names into the test name space. We will address This problem once the present task is complete (Issue#34)

If you are not acquainted with [check.v1](https://labix.org/gocheck) and some of the code looks odd, please ping me for help.

### CODE REVIEW

File: govcd/task_test.go [hunk 1]
```diff
@@ -5,22 +5,19 @@
...
+	"fmt"
 	. "gopkg.in/check.v1"
 )
 
-func (vcd *TestVCD) Test_WaitTaskCompletion(c *C) {
+func (vcd *TestVCD) Test_WaitTaskCompletion(check *C) {
 
-	testServer.Response(200, nil, taskExample)
+	fmt.Printf("Running: %s\n", check.TestName())
+	check.Skip("Disabled: need a reliable way of triggering a task")
```
Comment (auppunda) [inline govcd/task_test.go]: We can trigger a task with calling a method that creates a task for you.

**Comment by dataclouder** [comment]
Updated PR with vdc test and minor changes to catalog item test

File: govcd/vdc_test.go [hunk 2]
```diff
@@ -5,80 +5,97 @@
 package govcd
 
 import (
-	"github.com/vmware/go-vcloud-director/testutil"
-	types "github.com/vmware/go-vcloud-director/types/v56"
+	"github.com/vmware/go-vcloud-director/types/v56"
 	. "gopkg.in/check.v1"
-	"strconv"
+	// "strconv"
```
Comment (auppunda) [inline govcd/vdc_test.go]: delete strconv

File: govcd/vdc.go [hunk 3]
```diff
@@ -268,6 +268,7 @@ func (v *Vdc) ComposeVApp(orgvdcnetworks []*types.OrgVDCNetwork, vapptemplate VA
 				Info: "Configuration parameters for logical networks",
 			},
 		},
+		AllEULAsAccepted: true,
```
Comment (auppunda) [inline govcd/vdc.go]: should pass this as a parameter

**Comment by auppunda** [comment]
LGTM.

**Pull request merged**